resources/views/vendor/installer/environment-wizard.blade.php

```diff
@@ -512,7 +504,31 @@
 @endsection
 
 @section('scripts')
+<script src="http://ajax.googleapis.com/ajax/libs/jquery/1.7.1/jquery.min.js" type="text/javascript"></script>
     <script type="text/javascript">
+        $("#setup").on('submit',function(e) {
+        $("#submitme").attr("disabled", true);
+        $('#submitme').val('Installing, please wait...');
+        })
+
+
+        $('#app_name').keyup(function(e){
+            if(e.keyCode == 32) {
```

Comment:
what is this for?

Reply:
Space bar ASCII code is 32. We are disallowing space in App name in installer as .env don't accept those

Reply:
what if someone presses tab?

Reply:
We should have a backend validation for this instead. This can be bypassed very easily. Someone can copy paste an APP Name with spaces in it